_posts/2019-09-14-volunteer-opportunities.md

```diff
@@ -0,0 +1,32 @@
+---
+author: Rebecca Kindschi
+category: General
+date: 2019-09-14 06:00:00
+layout: post
+image: /static/img/blog/volunteer-opportunities-2019.png
+title: "Help Run DjangoCon US"
+---
+
+We have a wide variety of volunteer slots available! If you’d like to sign up for one (or more) we have a [handy spreadsheet](https://docs.google.com/spreadsheets/d/1VO8Xh6xPOQb32WjUD7pKJCMXvqCdcMz95CqIoNToUsA/edit#gid=0). Here's more information on each role.
+
+## Session Chairs
+
+The organizing committee is looking for folks to serve as **session chairs**.
```

Comment:
Maybe we can just delete this line? Seems strange now that I see it without all the suggestions around it.

Reply:
Good call, removed! Thanks!